src/react-components/2d-hud.js

```diff
@@ -231,7 +232,7 @@ class TopHUD extends Component {
     const tipDivForType = (type, cancelFunc) => (
       <div className={cx(styles.topTip)}>
         {cancelFunc && (
-          <button className={styles.tipCancel} onClick={cancelFunc}>
+          <button className={styles.tipCancel} aria-label="close" onClick={cancelFunc}>
```

Comment:
I think at some point we'll want to go back and get the `aria-labels` (and as much of everything else as we can) from a translations file.

Reply:
Yeah, that's fair. I think it makes sense to do so when we do the larger redesign, since a lot of these buttons ought to be abstracted away behind common components.